browser_linux.go

```diff
@@ -13,6 +16,19 @@ func Open(u string) (err error) {
 	if err != nil {
 		return
 	}
+
+	// read /proc/version which contains OS version info
+	data, err := ioutil.ReadFile("/proc/version")
```

Comment:
/proc/version is unavailable on macOS, so introducing it here will break macOS compatibility. I don't think /proc/version is part of LSB spec for comformance so we may also break bacwards compatibility on other systems.

However, `uname` is considered part of the LSB 5.0 spec as per page 13 of http://refspecs.linuxfoundation.org/LSB_5.0.0/LSB-Common/LSB-Common.pdf

What do you think about interacting with that command? It should be available and IIRC it also returns "Microsoft" when you do a `uname -s` or `uname -a`.

Reply:
Isn't this what the conditional compilation `// +build linux`  does?

Using `syscall.Uname` or `x/sys/unix.Uname` breaks on WSL, so we need to exec out and parse the ouput.

Reply:
Oh right, I forgot there's a browser_darwin.go. I really need to inspect all the code before commenting :headdesk:

lemme test on other OSes and see if this works across a few